cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseVersion.java

```diff
@@ -72,45 +75,13 @@ public String getMajorVersion() {
 
   private static Version currentVersion;
   private static String versionString;
+
   static {
```

Comment:
This static block is no longer needed, because of the new static method `determineVersion()`, which is guaranteed to be called to assign value to `currentVersion` before `currentVersion` is accessed.